src/searchset.ts

```diff
@@ -13,20 +17,21 @@ export interface SearchBundleEntry {
 }
 
 export class SearchSet {
-  // static attributes
   resourceType = 'Bundle';
   type = 'searchset';
   total: number;
   entry: SearchBundleEntry[] = [];
 
-  constructor(response: SearchResponse) {
-    // TO-DO Access list of search results & total result count from object defined in searchresponse.ts
-    this.total = response.totalCount;
+  constructor(response: JSON) {
+    this.total = 0; // TO-DO: Set the total number of trials returned by the match service
+    const trials = []; // TO-DO: Create an iterable version of the trials returned from the raw JSON of the match service (if necessary)
```

Comment:
I believe the trial objects need to be typed somehow

Reply:
yeah, I wasn't quite sure what to put there. That's how the raw json was typed in the old searchresponse file, so I just carried it over. By definition we can't know what the actually object will be since it will be different for each matcher, so I guess the question is how to properly handle incoming json. Should it just be of type any?

Reply:
any should be the type for now.

Reply:
changed type to any